### PR TITLE
Fix minor formatting issues

### PR DESCRIPTION
**Pull request by sigalsax**
### What does this PR do?
Fix minor formatting in testing files and update log message when unable to fetch credentials. 

### What ticket does this PR close?
Resolves -

#### Change log
- [ ] The CHANGELOG has been updated, or
- [x] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [x] The changes in this PR do not require tests

#### Documentation
- [ ] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [x] This PR does not require updating any documentation

### CODE REVIEW

File: conjur/errors_messages.py [hunk 1]
```diff
@@ -19,4 +19,4 @@
 MISMATCH_HOSTNAME_MESSAGE = "The machine's hostname did not match any names on the certificate. " \
                             "Make sure the names on the certificate (common name or SANs) match the machine's hostname."
 
-FETCH_CREDENTIALS_FAILURE_MESSAGE = "Failed to fetch credentials"
+FETCH_CREDENTIALS_FAILURE_MESSAGE = "Failed to fetch credentials. Log in again and try again."
```
Comment (eranha) [inline conjur/errors_messages.py:22]: `Log in again and try again."`?

File: conjur/wrapper/keystore_adapter.py [hunk 2]
```diff
@@ -65,15 +65,17 @@ def get_keyring_name(cls):
...
         """
         Method to check if the keyring is accessible
         """
-        # TODO investigate other errors that would make a keyring not accessible
         try:
-            # Send a dummy request to test if the keyring is accessible
+            # The following sends a dummy request to test if the system's keyring is
+            # accessible. 'get_password' returns None if the password does not exist.
+            # If get_password raises an error, then we can infer that the keyring has
+            # not be configured correctly and not available for use and return false
```
Comment (eranha) [inline conjur/wrapper/keystore_adapter.py:77]: If get_password raises an error, then we can infer that the keyring has
not **been** configured correctly and not available for use and return false

there is A and B and C in the above sentence

File: conjur/errors_messages.py (hunk 1 above)
Comment (sigalsax) [reply conjur/errors_messages.py:22]: @shulifink approval need here as well

Comment (sigalsax) [reply conjur/errors_messages.py:22]: @shulifink reminder

Comment (sigalsax) [reply conjur/errors_messages.py:22]: Log in again and re-run the command @shulifink

**Review by mbenita-Cyberark — DISMISSED** [review]

**Review by mbenita-Cyberark — APPROVED** [review]

**Pull request merged**